src/librustc/ty/util.rs

```diff
@@ -16,7 +16,7 @@ use infer::InferCtxt;
 use ich::{StableHashingContext, NodeIdHashingMode};
 use traits::{self, Reveal};
 use ty::{self, Ty, TyCtxt, TypeFoldable};
-use ty::ParameterEnvironment;
+use ty::ParamEnv;
```

Comment:
This is again somewhat painful to me since we use both `ty::ParamEnv` and `ParamEnv` in this file. Don't bother fixing if you don't want to though.